docs/DeveloperGuide.adoc

```diff
@@ -193,14 +193,15 @@ image::LogicClassDiagram.png[width="800"]
 link:{repoURL}/src/main/java/seedu/address/logic/Logic.java[`Logic.java`]
 
 .  `Logic` uses the `OrderBookParser` class to parse the user command.
+.  `OrderBookParser` further creates either a `OrderCommandParser` or `DeliverymanCommandParser` to parse order or deliveryman command respectively. Other commands that is neither order or deliveryman skip this creation of the extra Parse
```

Comment:
"... creation of the extra Parser"

Reply:
you can suggest this as a change and make it easier for him to edit 👍